modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsa.ts

```diff
@@ -54,18 +54,7 @@ export class EcdsaUtils extends baseTSSUtils<KeyShare> {
     const userKeyShare = await MPC.keyShare(1, m, n);
     const backupKeyShare = await MPC.keyShare(2, m, n);
 
-    const randomHexString = crypto.randomBytes(12).toString('hex');
-    openpgp.config.rejectCurves = new Set();
-
-    const userGpgKey = await openpgp.generateKey({
-      userIDs: [
-        {
-          name: randomHexString,
-          email: `user-${randomHexString}@${randomHexString}.com`,
-        },
-      ],
-      curve: 'secp256k1',
-    });
+    const userGpgKey = await generateGPGKeyPair('secp256k1');
```

Comment:
Missing username and email?

Reply:
user name and email parameters are optional, if they are missing they are generated randomly inside this function.
In all places in the current PR the values for both name and email were randomly generated and followed the same format so it is not necessary to explicitly provide them.

Reply:
Awesome, thanks!